exporters/stdout/stdoutlog/exporter_test.go

```diff
@@ -274,3 +275,60 @@ func getRecord(now time.Time) sdklog.Record {
 
 	return record
 }
+
+func TestExporterConcurrentSafe(t *testing.T) {
+	testCases := []struct {
+		name     string
+		exporter *Exporter
+	}{
+		{
+			name:     "zero value",
+			exporter: &Exporter{},
+		},
+		{
+			name: "new",
+			exporter: func() *Exporter {
+				exporter, err := New()
+				require.NoError(t, err)
+				require.NotNil(t, exporter)
+
+				return exporter
+			}(),
+		},
+	}
+
+	for _, tc := range testCases {
+		t.Run(tc.name, func(t *testing.T) {
+			exporter := tc.exporter
+
+			runConcurrently(
+				func() {
+					err := exporter.Export(context.Background(), []sdklog.Record{{}})
+					assert.NoError(t, err)
+				},
+				func() {
+					err := exporter.ForceFlush(context.Background())
+					assert.NoError(t, err)
+				},
+				func() {
+					err := exporter.Shutdown(context.Background())
+					assert.NoError(t, err)
+				},
+			)
```

Comment:
This does not check running `Shutdown` (and other methods) concurrently many times.

```suggestion
			const goroutines = 5
			var wg sync.WaitGroup
			wg.Add(goroutines)
			for i := 0; i < goroutines; i++ {
					go func() {
							defer wg.Done()
							err := exporter.Export(context.Background(), []sdklog.Record{{}})
							assert.NoError(t, err)
							err = exporter.ForceFlush(context.Background())
							assert.NoError(t, err)
							err := exporter.Shutdown(context.Background())
							assert.NoError(t, err)
					}()
			}
			wg.Wait()
```